suzieq/gui/stlit/xplore.py

```diff
@@ -114,10 +115,10 @@ def _create_sidebar(self) -> None:
             state.namespace = namespace
 
         if state.table:
-            tables_obj = get_sqobject('tables')(start_time=state.start_time,
+            state.tables_obj = get_sqobject('tables')(start_time=state.start_time,
```

Comment:
You can use the describe function of the table itself. So you can do `get_sqobject(state.table)().describe()`. I'm going to deprecate `table describe command=` at some point.